docs/guides/core-concepts/cypress-app.mdx

```diff
@@ -191,11 +191,12 @@ without leaving the Cypress app.
 This feature eliminates the tedious process of switching between Cypress, your
 local editor, and CI output, in order to fix failed tests. It does this by
 enabling you to use the [Test Runner](#Test-Runner) to run only the tests that
-failed in your last recorded test run, as well as review screenshots, videos,
-and logs from your tests.
+failed in your last recorded test run, as well as review the
+[Test Replay](/guides/cloud/debugging/test-replay) and other artifacts like screenshots,
+videos, and logs from your tests.
 
 The Debug page shows the latest completed
-[test run](/guides/cloud/recorded-runs#Latest-Runs) that matches the HEAD commit
+[test run](/guides/cloud/debugging/recorded-runs#Latest-Runs) that matches the HEAD commit
```

Comment:
```suggestion
[test run](/guides/cloud/debugging/recorded-runs#Latest-Runs) that matches the `HEAD` commit
```

Reply:
This is just linting but figured it looks a bit clearer? not a needed change by any means